.storybook/webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = (config, env) => {
+  config.module.rules = [
+    ...config.module.rules,
+    { test: /\.(gif|svg|jpg|png)$/, loader: 'file-loader' },
+  ];
+  return config;
+};
```

Comment:
this webpack config is only used by storybook. had to include `file-loader` manually and update its version so images can be loaded properly.